UnitTests/StreamWithTimeoutDecoratorTest.cs

```diff
@@ -292,57 +292,57 @@ public void WriteByte_IsNotOverridden ()
     [Test]
     public void Write_InnerCannotTimeout ()
     {
-      _innerStreamMock.Expect (s => s.CanTimeout).Return (false);
+      _innerStreamMock.Setup (s => s.CanTimeout).Returns (false);
       byte[] buffer = new byte[10];
-      _innerStreamMock.Expect (s => s.Write (buffer, 22, 33));
+      _innerStreamMock.Setup (s => s.Write (buffer, 22, 33));
 
       _decorator.Write (buffer, 22, 33);
-      _innerStreamMock.VerifyAllExpectations();
+      _innerStreamMock.VerifyAll();
     }
 
     [Test]
     public void Write_UnderlyingStreamTimesOut_ThrowsTimeoutException ()
     {
-      _innerStreamMock.Expect (s => s.CanTimeout).Return (true);
+      _innerStreamMock.Setup (s => s.CanTimeout).Returns (true).Verifiable();
       byte[] buffer = new byte[10];
 
-      var waitHandle = MockRepository.GenerateStrictMock<WaitHandle>();
-      waitHandle.Expect (w => w.WaitOne (TimeSpan.FromMilliseconds (123))).Return (false);
+      var waitHandle = new Mock<WaitHandle> (MockBehavior.Strict); //REVIEW strict hier sinnvoll? mock wird später nicht einmal verified
```

Comment:
```suggestion
      var waitHandle = new Mock<WaitHandle> (MockBehavior.Strict);
```

Reply:
In a first step we're trying to 1:1 map the old syntax to the new syntax. Semantic cleanup will be done in another step.